server/endpoints/models/Client.js

```diff
@@ -0,0 +1,64 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const Location = new Schema({ 
+    city: String, 
+    state: String 
+})
+
+const ClientSchema = new Schema({
+  email: {
+    type: String,
+  },
+  name: {
+    type: String,
+  },
+  language: {
+    type: String,
+  },
+  bio: {
+    type: String,
+  },
+  postiveRatingCount: {
+    type: Number
+  },
+  negativeRatingCount: {
+    type: Number
+  },
+  location: {
```

Comment:
did you want to use GeoJSON for location?

Reply:
Yes, I think that would be great.

Reply:
This change is in `main` now.